googleapiclient/discovery.py

```diff
@@ -201,10 +201,8 @@ def build(serviceName,
   if resp.status >= 400:
     raise HttpError(resp, content, uri=requested_url)
 
-  try:
+  if isinstance(content, bytes):
     content = content.decode('utf-8')
-  except AttributeError:
-    pass
```

Comment:
I think `http.request` always return contents as `bytes`.
So we can just remove this `try`~`except`.
Or `assert isinstance(content, bytes)`.